controllers/suite_test.go

```diff
@@ -71,6 +80,21 @@ var _ = BeforeSuite(func(done Done) {
 	Expect(err).ToNot(HaveOccurred())
 	Expect(k8sClient).ToNot(BeNil())
 
+	ctx = context.Background()
+
```

Comment:
We have `k8sClient` initiated at this time. You can then load secrets like:
https://github.com/stakater/jira-service-desk-operator/blob/d79841c9afa3de4d9d1364dbe34b56d796c56bf8/pkg/jiraservicedesk/config/config.go#L56

But instead of `LoadSecretData` use `LoadSecretDataUsingClient` method.

Reply:
For `LoadSecretDataUsingClient` update operator-utils to v0.1.4 in `go.mod`